xbee/__init__.py

```diff
@@ -9,6 +9,6 @@
 __author__ = 'n.io'
 __license__ = 'MIT'
 
-from xbee.ieee import XBee
-from xbee.zigbee import ZigBee
-from xbee.digimesh import DigiMesh
+from xbee.thread import XBee
```

Comment:
No.  The backend bits are shared functionality, and is not an implementation ready for use.  The threaded implementation is the default, what exists in master now, and aims to keep backwards compatibility.